tests/pytorch_pfn_extras_tests/training_tests/extensions_tests/test_print_report.py

```diff
@@ -3,18 +3,29 @@
 import pytest
 
 import pytorch_pfn_extras as ppe
-
-
-def test_run_print_report():
+from pytorch_pfn_extras.training import extensions
+
+
+@pytest.mark.parametrize(
+    'format,append',
+    [
+        ('json', False),
+        ('json-lines', True),
+        ('json-lines', False),
+        ('yaml', True),
+        ('yaml', False),
+    ]
+)
+def test_run_print_report(format, append):
     max_epochs = 5
     iters_per_epoch = 5
     manager = ppe.training.ExtensionsManager(
         {}, {}, max_epochs, iters_per_epoch=iters_per_epoch)
 
     out = io.StringIO()
-    log_report = ppe.training.extensions.LogReport()
+    log_report = extensions.LogReport(format=format, append=append)
     manager.extend(log_report)
-    extension = ppe.training.extensions.PrintReport(out=out)
+    extension = extensions.PrintReport(out=out)
```

Comment:
I think this is not testing that the file is being written.
but it wasn't doing it before either. So we should do it in a different PR.